content/practices/outcome-oriented-roadmap/index.md

```diff
@@ -0,0 +1,225 @@
+---
+# remove "draft:true" when ready for final publication 
+draft: false
+title: "Outcome-Oriented Roadmap"
+linkTitle: "Outcome Oriented Roadmap"
+description: "A communication and planning tool that shows sequential outcomes that help execute your strategy and build towards your vision."
+tags: ["Inception"]
```

Comment:
I'm not sure Inception is the right tag here. A roadmap usually emerges during product development and is updated throughout. I've never done a roadmap in an Inception. Maybe during Discovery and/or Framing.

Reply:
Updated to be Discovery and Framing, removed Inception.